{"name": "test_set_comparison", "status": "failed", "statusDetails": {"message": "AssertionError: assert {'0', '2'} == {'0', '1', '2'}\n  Extra items in the right set:\n  '1'\n  Full diff:\n  - {'2', '0', '1'}\n  ?          -----\n  + {'2', '0'}", "trace": "def test_set_comparison():\n        set1 = set(\"2021\")\n        set2 = set(\"2020\")\n        # pdb.set_trace()\n>       assert set2 == set1\nE       AssertionError: assert {'0', '2'} == {'0', '1', '2'}\nE         Extra items in the right set:\nE         '1'\nE         Full diff:\nE         - {'2', '0', '1'}\nE         ?          -----\nE         + {'2', '0'}\n\nPython_Pytest\\com\\kxj\\case\\test_04_exception.py:13: AssertionError"}, "start": 1617330625666, "stop": 1617330625668, "uuid": "12f94ce0-a715-4107-ae8e-ed2a4e7ee3cb", "historyId": "cf6a8606e27ffddb107586b2d1b346f1", "testCaseId": "caf7234ca5659266a8e5a3907414e85f", "fullName": "com.kxj.case.test_04_exception#test_set_comparison", "labels": [{"name": "parentSuite", "value": "com.kxj.case"}, {"name": "suite", "value": "test_04_exception"}, {"name": "host", "value": "DESKTOP-4IJL66C"}, {"name": "thread", "value": "3396-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "com.kxj.case.test_04_exception"}]}